tests/person.test.js

```diff
@@ -0,0 +1,49 @@
+import { mount, createLocalVue } from '@vue/test-utils'
+import Vuex from 'vuex'
+
+import Person from '@/components/Person'
+
+const localVue = createLocalVue()
+
+localVue.use(Vuex)
+
+const results = {
+  'name': 'Darth Vader',
+  'height': '202',
+  'mass': '136',
+  'hair_color': 'none',
+  'skin_color': 'white',
+  'eye_color': 'yellow',
+  'birth_year': '41.9BBY',
+  'gender': 'male',
+  'homeworld': {'name': 'Tatooine', 'url': 'https://swapi.co/api/planets/1/'},
+  'species': 'Human',
+  'vehicles': [],
+  'starships': [{'name': 'TIE Advanced x1', 'url': 'https://swapi.co/api/starships/13/'}]
+}
+
+const state = {
+  'results': results
+}
+
+var store = new Vuex.Store({
```

Comment:
Not sure why we're creating a new store here instead of just importing the actual store like in `home.test.js`.